cmd/podman/containers/rm.go

```diff
@@ -108,10 +106,21 @@ func rm(cmd *cobra.Command, args []string) error {
 			if errors.Cause(err) == define.ErrWillDeadlock {
 				logrus.Errorf("Potential deadlock detected - please run 'podman system renumber' to resolve")
 			}
+			setExitCode(r.Err)
 			errs = append(errs, r.Err)
 		} else {
 			fmt.Println(r.Id)
 		}
 	}
 	return errs.PrintErrors()
 }
+
+func setExitCode(err error) {
+	cause := errors.Cause(err)
+	switch cause {
+	case define.ErrNoSuchCtr:
+		registry.SetExitCode(1)
+	case define.ErrCtrStateInvalid:
+		registry.SetExitCode(2)
+	}
+}
```

Comment:
do we need to set the registry.ExitCode to something by default if it's not one of these two?  Unknown perhaps?

Reply:
@TomSweeneyRedHat Yes, it is initialized to 125 when created.